ocp_resources/volume_snapshot.py

```diff
@@ -1,4 +1,4 @@
-from ocp_resources.resource import NamespacedResource
```

Comment:
Can you please split the 2 classes to files (in another PR please)

Reply:
Sure